apps/settings/js/modules/panel_utils.js

```diff
@@ -270,15 +270,13 @@ define(function(require) {
               spanFields[i].textContent = result[key];
             }
           } else { // result[key] is undefined
-            var _ = navigator.mozL10n.get;
             switch (key) {
               //XXX bug 816899 will also provide 'deviceinfo.software' from
               // Gecko which is {os name + os version}
               case 'deviceinfo.software':
                 navigator.mozL10n.setAttributes(spanFields[i],
                   'deviceInfo_software',
-                  { brandShortName: _('brandShortName'),
-                    os: result['deviceinfo.os'] });
+                  { os: result['deviceinfo.os'] });
```

Comment:
Hahah, no! Sorry, the patch makes it unclear what happens here.

So, basically right now in .properties we have the dual-nature of placeables.

```
foo = {{ brandShortName }} loves you
```
will take brandShortName from a variable (l10n-args) or if there will not be such a key it will take another entity's value and in this case, we have an entity
```
brandShortName=Firefox OS
```

so it'll just call it in `deviceInfo_software` entity.

So basically this argument was not needed, because localization knows you want the value of an entity `brandShortName`. :)